platform/provider.go

```diff
@@ -90,6 +90,8 @@ func NewProvider(logger boshlog.Logger, dirProvider boshdirs.Provider, statsColl
 		devicePathResolver = devicepathresolver.NewVirtioDevicePathResolver(idDevicePathResolver, mappedDevicePathResolver, logger)
 	case "scsi":
 		devicePathResolver = devicepathresolver.NewScsiDevicePathResolver(500*time.Millisecond, fs)
+	case "scsiid":
+		devicePathResolver = devicepathresolver.NewScsiIDDevicePathResolver(50000*time.Millisecond, fs, logger)
```

Comment:
Interesting, is this the only change for the agent? What else is different between this and vsphere stemcell?